Kernel/FileSystem/Ext2Base.h

```diff
@@ -0,0 +1,307 @@
+/*
+ * Copyright (c) 2022, iProgramInCpp <iprogramincpp@gmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+namespace Kernel::Ext2 {
+// The second extended filesystem constants/structures
+
+constexpr u32 ext4_feature_incompat_64bit = (1 << 7);
+constexpr u32 max_name_len = 255;
+constexpr u16 super_magic = 0xEF53;
+
+// Constants relative to the data blocks
```

Comment:
Do we want dots after comments in cases like this, or not? Same for all the other comments.